### ISSUE TITLE
"Remaining days" isn't accurate at all

### ISSUE THREAD

**Issue by frdmn** [frdmn/openssl-cert-tools#3]
```javscript
var opensslTools = require('./main');

var demoCertificate =
'-----BEGIN CERTIFICATE-----\n' +
'MIIEuzCCA6OgAwIBAgIDAlzmMA0GCSqGSIb3DQEBCwUAMEcxCzAJBgNVBAYTAlVT\n' +
'MRYwFAYDVQQKEw1HZW9UcnVzdCBJbmMuMSAwHgYDVQQDExdSYXBpZFNTTCBTSEEy\n' +
'NTYgQ0EgLSBHMzAeFw0xNTAyMTUxMDA5MzhaFw0xNjA0MTgwNzE3NDFaMIGWMRMw\n' +
'EQYDVQQLEwpHVDc0MzM3NDEyMTEwLwYDVQQLEyhTZWUgd3d3LnJhcGlkc3NsLmNv\n' +
'bS9yZXNvdXJjZXMvY3BzIChjKTE1MS8wLQYDVQQLEyZEb21haW4gQ29udHJvbCBW\n' +
'YWxpZGF0ZWQgLSBSYXBpZFNTTChSKTEbMBkGA1UEAxMSd3d3Lndlcm5lci1rZXJu\n' +
'LmRlMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAyU9p0Y5Yx1/tgzPB\n' +
'Cbzq76S+ff/LFYPgBmD9qiOLGs9KW/NEWqkNsmITclKQezv1ZqXDQw01IJzvlUG1\n' +
'XJzX2oUfCBhGdFtk3MBMnpk5oQkJ8/zyg7fVEsbCceGSvtSWWkXc9GZ8bRb49p9p\n' +
'raheiJRDKgovMQIUh6EH3YP0kzHQ6YY7EMx5RYYT1yIZXIOgb2gaw+wni9yNguET\n' +
'YBGhhoxdP630O04Oy2fwwSAxi7D2fHjnHbOLXfmVN1mn8gwEkl9INilh4GDuGQI/\n' +
'+KG0hiGuq3DiVkBMdzs+E8jlDqTr0xL/VmbKU7gRDbMTXuzYmac3te60bgyLKXJR\n' +
'WeP/WQIDAQABo4IBXjCCAVowHwYDVR0jBBgwFoAUw5zz/NNGCDS7zkZ/oHxb8+II\n' +
'y1kwVwYIKwYBBQUHAQEESzBJMB8GCCsGAQUFBzABhhNodHRwOi8vZ3Yuc3ltY2Qu\n' +
'Y29tMCYGCCsGAQUFBzAChhpodHRwOi8vZ3Yuc3ltY2IuY29tL2d2LmNydDAOBgNV\n' +
'HQ8BAf8EBAMCBaAwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMC0GA1Ud\n' +
'EQQmMCSCEnd3dy53ZXJuZXIta2Vybi5kZYIOd2VybmVyLWtlcm4uZGUwKwYDVR0f\n' +
'BCQwIjAgoB6gHIYaaHR0cDovL2d2LnN5bWNiLmNvbS9ndi5jcmwwDAYDVR0TAQH/\n' +
'BAIwADBFBgNVHSAEPjA8MDoGCmCGSAGG+EUBBzYwLDAqBggrBgEFBQcCARYeaHR0\n' +
'cHM6Ly93d3cucmFwaWRzc2wuY29tL2xlZ2FsMA0GCSqGSIb3DQEBCwUAA4IBAQCD\n' +
'Aq1uIH6WIcDtLzjem9cQJcyjIezKpZaFpiT9RPfGy8UCaZa3K9WZF3hQFFaEZU9T\n' +
'1SlgHvSukhA27hHiSD+ZHpY54WCEBxEk/MgopYU568cwVJ7M0dR0NJEB40g+2/ZJ\n' +
'y0qdFH/ooEjshSBwwjLAN6XWR1ZzacxCSsXxJMz5GvYSkNcmElemhOGoNRY0doXz\n' +
'ZfzCDfN2yIQPlc5xdOVud72Dv6usUqq/kLfEXSURxqVc1tG0sRQvM0fC9/V3ACB/\n' +
'yeJeOHbFhdDy/ucj2i2ktVlpzmSddY1dfQtZNEQMpDsfs2PsI4H1wJse+dAxL6gi\n' +
'04vx08Z1zyMGXPo4ZYnl\n' +
'-----END CERTIFICATE-----';

opensslTools.getCertificateInfo(demoCertificate, function(err, data){
  if (err) {
    console.log(err)
  } else {
    console.log(data);
    /* validFrom: Sun Feb 15 2015 11:09:38 GMT+0100 (CET),
     * validTo: Mon Apr 18 2016 09:17:41 GMT+0200 (CEST),
     * remainingDays: 428
     * => 🖕
     */
  }
});
```

**Comment by frdmn** [comment]
from Feb 15 to Apr 18 are no 428 days ...

**Comment by frdmn** [comment]
Fixed in 2216c68c627698a33f46483c1cc7eadff2cfc0c8
